godel_path_planning/src/trajectory_planning.cpp

```diff
@@ -37,8 +39,34 @@ namespace
     return ptr;
   }
 
+  descartes_core::TrajectoryPtPtr rotationalMultipoint(double x, double y, double z, double rx, double ry, double rz, double plus_minus,
+                                                       double discretization, const descartes_core::TimingConstraint& timing)
+  {
+    using namespace descartes_core;
+    using namespace descartes_trajectory;
+
+    CartTrajectoryPt a(
+              TolerancedFrame(utils::toFrame(x,y, z, rx, ry, rz, descartes_core::utils::EulerConventions::XYZ),
+               ToleranceBase::zeroTolerance<PositionTolerance>(x, y, z),
+               ToleranceBase::createSymmetric<OrientationTolerance>(rx, ry, rz-plus_minus, 0, 0, 0.3)),
+              0.0, discretization, timing);
+
+    
+    CartTrajectoryPt b(
+              TolerancedFrame(utils::toFrame(x,y, z, rx, ry, rz, descartes_core::utils::EulerConventions::XYZ),
+               ToleranceBase::zeroTolerance<PositionTolerance>(x, y, z),
+               ToleranceBase::createSymmetric<OrientationTolerance>(rx, ry, rz+plus_minus, 0, 0, 0.3)),
+              0.0, discretization, timing);
+
+    std::vector<CartTrajectoryPt> pts;
+    pts.push_back(a); pts.push_back(b);
+
+    return descartes_core::TrajectoryPtPtr(new godel_path_planning::CartTrajectoryMultiPt(pts, timing));
+
+  }
+
   // Create an axial trajectory pt from a given tf transform
-  descartes_core::TrajectoryPtPtr tfToAxialTrajectoryPt(const tf::Transform& nominal, double discretization)
+  descartes_core::TrajectoryPtPtr tfToAxialTrajectoryPt(const tf::Transform& nominal, double discretization, double z_offset, bool free_z)
```

Comment:
It doesn't look like the **z_offset** parameter is used for anything.  If so it should probably be removed.